File: example/optimizely/js/sample-OPT.ts
FeatureFlag.start("your_env_id", "your_api_key");

const visitor = FeatureFlag.newVisitor({
    visitorId: "your_visitor_id",
    context: { isVip: true },
});

visitor.on("ready",  (error) => {
    if (error) {
        return;
    }

    const btnColorFlag: string = client.getFeatureVariable('my_feature_key', 'OPT-flag-ts', 'user_123', attributes);

});

visitor.fetchFlags();

File: cmd/analyze/competitor-example/optimizely/net/sample-OPT.cs
var visitor = Ff.NewVisitor("<VISITOR_ID>")
  .IsAuthenticated(true)
  .HasConsented(true)
  .WithContext(new Dictionary<string, object> {
    ["isVIP"] = true,
    ["country"] = "NL",
    ["loginProvider"] = "Google"
    })
  .Build();

  await visitor.FetchFlags();

var flag = visitors.GetFeatureVariableDouble("my_feature_key", "OPT-flag-cs", "user_123", attributes);
